components/com_content/Model/ArticlesModel.php

```diff
@@ -484,8 +481,8 @@ function ($data) use ($db)
 		switch ($dateFiltering)
 		{
 			case 'range':
-				$startDateRange = $db->quote($this->getState('filter.start_date_range', $nullDate));
-				$endDateRange   = $db->quote($this->getState('filter.end_date_range', $nullDate));
+				$startDateRange = $db->quote($this->getState('filter.start_date_range'));
```

Comment:
@wilsonge Here and in the following line you have to advise me if that is ok or if there has to be something done for one of the range start and end dates being not specified, or both even. The existing code might do the wrong thing in such case, if that case is allowed.

Reply:
what are the options for this? We've effectively mandated `a.created` and `a.modified`

Reply:
no, these here are start and end dates for a range filtering, i think. they are entered in a filter. the date to be fitered for is either created or modified, but my question above refers to that start and end dates of the range for filtering. or am i getting something wrong?

Reply:
ok can be `a.publish_up` (mod_articles_popular and mod_articles_category). Just need to test this still works